tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -304,6 +363,20 @@ Object.PCM {
 			formats	'S16_LE,S24_LE,S32_LE'
 		}
 	}
+	pcm."98" {
+		name	"DMIC Raw 1"
+		id	98
```

Comment:
please not use 98, use 18 which is a defined number for all audio framework like pulse audio.

Reply:
sure, will change it to 18.